app/routes/index.js

```diff
@@ -28,6 +28,22 @@ const getModel = (data, question) => {
   return model
 }
 
+const customiseErrorText = (value, currentQuestion, errorList, errorText, yarKey, h) => {
+  const baseModel = getModel(value, currentQuestion)
```

Comment:
instead of having different argumens we can just have currentQuestion and errorList and h , and get everything else from currentQuestion inside of this function and use them